### PR TITLE
VAGOV-6065: Adding user_history module and patch.

### PR DESCRIPTION
**Pull request by ethanteague**
## What this does
 - Adds a user history tab to user pages for tracking user creation and editing events
 - Adds a user history list page: `/user_history/list`

## QA / Installation
 - Go to `/user_history/initialise` or visit the status report and click on the link.
 - Visit user edit page, look for history tab.
 - Visually verify that creation / editing events are being logged.
 - Go to `user_history/list` and visually verify that all users and account history appears

### CODE REVIEW

**Comment by kevwalsh** [comment]
@ElijahLynn can you look at this phpunit fail? looks like a blip to me.

**Comment by ElijahLynn** [comment]
It is accurate it appears. Need to enable module in config. 

> The module user_history does not exist.

File: composer.json [hunk 1]
```diff
@@ -240,6 +241,9 @@
             "drupal/textfield_counter": {
                 "Prevent form submission when character limit exceeded option does not work": "https://www.drupal.org/files/issues/2018-10-04/textfield_counter-prevent_form_submission_does_not_work-3004457-2.patch"
             },
+            "drupal/user_history": {
+                "Invalid argument in foreach when saving a user": "https://www.drupal.org/files/issues/2019-12-11/invalid-argument-on-save_0.patch"
```
Comment (swirtSJW) [inline composer.json]: I checked out the Patch.   Looks good.

File: config/sync/core.extension.yml [hunk 2]
```diff
@@ -152,6 +152,7 @@ module:
   toolbar_menu: 0
   update: 0
   user: 0
+  user_history: 0
```
Comment (swirtSJW) [inline config/sync/core.extension.yml]: Shouldn't user_created_by show up here too?

File: composer.json [hunk 3]
```diff
@@ -79,6 +79,7 @@
         "drupal/toolbar_menu_clean": "^1.0",
         "drupal/tour_ui": "^1.0@beta",
         "drupal/ui_patterns": "^1.0",
+        "drupal/user_history": "^1.0@alpha",
```
Comment (swirtSJW) [inline composer.json]: Should user_created_by be here too?

File: config/sync/core.extension.yml (hunk 2 above)
Comment (ethanteague) [reply config/sync/core.extension.yml]: user_created_by shouldn't be here - leftover from a previous project usage attempt. Removed, @swirtSJW

File: composer.lock [hunk 4]
```diff
@@ -19217,6 +19212,65 @@
                 "source": "https://git.drupalcode.org/project/media_entity_generic"
             }
         },
+        {
+            "name": "drupal/user_created_by",
```
Comment (swirtSJW) [inline composer.lock]: I think this needs to go away.

Comment (ethanteague) [reply composer.lock]: Nice catch, @swirtSJW :+1: - removed

**Pull request merged**